docs/rules/valid-typeof.md

```diff
@@ -28,6 +34,24 @@ typeof foo === baz
 typeof bar === typeof qux
 ```
 
+Examples of **incorrect** code with the `{ "requireStringLiterals": true }` option:
+
+```js
+typeof foo === undefined
+typeof bar == Object
+typeof baz === "strnig"
+typeof foo === 5
+typeof bar === typeof qux
+```
+
+Examples of **correct** code with the `{ "requireStringLiterals": true }` option:
+
+```js
+typeof foo === "undefined"
+typeof bar == "object"
+typeof baz === "string"
```

Comment:
Never mind, I guess we are checking that. In that case I would love the docs to mention this and the example I proposed should be listed in incorrect code.